2fa not provider (#4)
rullzer/blog: content/post/20181019-2FA-Notifications.md

---
title: "Two-Factor via Nextcloud Notifications"
date: "2018-10-19"
tags:
  - Nextcloud
  - Security
---

> I'm happy to announce a new two-factor provider for your Nextcloud. The

Reviewer: ```suggestion
I'm happy to announce a new two-factor provider for your Nextcloud: the
```

*Notifications Provider*. This provider utilizes your existing logged in
devices to grant new devices access to your Nextcloud.

> Note: the app is still under heavy development. Still we appreciate testing
> and feedback!

The flow is simple. You enable the provider in your personal security settings.

> Then the next time you login you can chose to authenticate using a device that

Reviewer: ```suggestion
Then the next time you log in you can chose to authenticate using a device that
```

is already logged in to your account.

{{% img-responsive "/static/images/posts/2FA-Notifications/login.png" %}}

> Now a notification is dispatched. This is delivered to all your devices. Which

Reviewer: ```suggestion
Now a notification is dispatched. This is delivered to all your devices, which
```

> means that you get even push notifications! That might just look something like:

Reviewer: ```suggestion
means that you even get push notifications! That might just look something like:
```

{{% img-responsive "/static/images/posts/2FA-Notifications/android_push.png" %}}

You can approve or cancel the login from any of your devices. If you approved
the login then you will be automatically logged in.

> You can grab the app from [appstore](https://apps.nextcloud.com/apps/twofactor_nextcloud_notification). And

Reviewer: ```suggestion
You can grab the app from the [app store](https://apps.nextcloud.com/apps/twofactor_nextcloud_notification). And
```
